db/schema.rb

```diff
@@ -234,6 +231,17 @@
     t.index ["tenant_id"], name: "index_service_plans_on_tenant_id"
   end
 
+  create_table "tag_links", force: :cascade do |t|
+    t.bigint "tenant_id"
+    t.bigint "order_process_id"
+    t.string "app_name"
+    t.string "object_type"
+    t.string "tag_name"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["app_name", "object_type", "tag_name"], name: "index_tag_links_on_app_name_and_object_type_and_tag_name", unique: true
```

Comment:
Did this index not get updated when you added the `tenant_id` to it in the migration?

Reply:
It should, fixed.